docs/editor/integrated-terminal.md

```diff
@@ -19,50 +19,62 @@ To open the terminal:
 
-## Managing multiple terminals
+## Managing terminals
 
-You can create multiple terminals open to different locations and easily navigate between them. Terminal instances can be added by clicking the plus icon on the top-right of the **TERMINAL** panel or by triggering the `kb(workbench.action.terminal.new)` command. This action creates another entry in the dropdown list that can be used to switch between them.
+The terminal tabs view is to the right of the terminals. Each terminal has an entry with its name, icon, color, and group decoration (if any).
 
-![Multiple Terminals](images/integrated-terminal/terminal-multiple-instances.png)
+![Terminal tabs](images/integrated-terminal/tabs.png)
+> **Tip:** Change the tabs location using the `integrated.terminal.tabs.location` setting.
+
+Terminal instances can be added by clicking the plus icon on the top-right of the **TERMINAL** panel, selecting a profile from the new terminal dropdown, or by triggering the `kb(workbench.action.terminal.new)` command. This action creates another entry in the tab list associated with that terminal.
+
+Remove terminal instances by hovering a tab and pressing the trash can button, selecting a tab item and pressing `kbstyle(delete)`, by triggering the `kb(workbench.action.terminal.kill)` command, or via the right click context menu.
+
+Navigate between terminal groups using focus next `kbstyle(workbench.action.terminal.focusNext)` and focus previous `kbstyle(workbench.action.terminal.focusPrevious)`.
+
+Icons may appear to the right of the terminal title on the tab label when the terminal's status change. Statuses include disconnected, in need of relaunch, and bell. Hover the icon to read status information and to take action.
 
-Remove terminal instances by pressing the trash can button.
+### Grouping
 
->**Tip:** If you use multiple terminals extensively, you can add key bindings for the `focusNext`, `focusPrevious` and `kill` commands outlined in the [Key Bindings section](/docs/editor/integrated-terminal.md#terminal-keybindings) to allow navigation between them using only the keyboard.
+Split the terminal by:
+- triggering the `kb(workbench.action.terminal.split)` command
+- right clicking the context menu and selecting the `Split...` submenu option
+- `kbstyle(alt)` clicking a tab, the plus button, or the the single tab on the terminal panel
 
-### Terminal Splitting
+Navigate between terminals in a group by focusing the previous pane, `kb(workbench.action.terminal.focusPreviousPane)`, and focusing the next pane, `kb(workbench.action.terminal.focusNextPane)`.
 
-You can also split the terminal by triggering the `kb(workbench.action.terminal.split)` command or via the right click context menu.
+Drag and drop tabs to re-arrange the groups.
 
-![Split terminals](images/integrated-terminal/terminal-split-pane.png)
+Unsplit a split terminal by triggering the `kb(workbench.action.terminal.unsplit)` command.
```

Comment:
Suggestion:

Tabs support drag and drop to allow re-arranging, dragging an entry in a terminal group into the empty will remove it from the group (ie. unsplit), dragging a tab into the main terminal area allows joining a group.

Can remove the command part imo